### DISCUSSION TITLE
find a little bug

### DISCUSSION THREAD

**Discussion by kewei19880602** [openai-php/client#157] [General]
sorry, i am chinese ,my english is poor.
thanks for this storehouse. this day , i don't know what happen,  once in a while openai will show this error
 PHP Warning:  Undefined array key "choices" in /vendor/openai-php/client/src/Responses/Chat/CreateStreamedResponse.php on line 44
and then, it will show fatal error array_map():Argument #($array) must be of type array, null given

so ,i think we need to throw a exception
if(!isset($attributes['choices'])) {
    throw new \Exception('');
}
thanks

**Comment by gehrisandro** [comment]
Hi @ZhangKoven 

Can you provide an example how to reproduce this error?
